README.md

```diff
@@ -26,7 +33,60 @@ struct Person {
 }
 
 fn main() {
-    let conn = Connection::connect("postgres://postgres@localhost", SslMode::None).unwrap();
```

Comment:
It's discouraged in "real" code, but used in examples to focus them on the APIs they're showing you.